homeassistant/components/lg_soundbar/config_flow.py

```diff
@@ -32,6 +35,7 @@ def msg_callback(response):
     try:
         connection = temescal.temescal(host, port=port, callback=msg_callback)
         connection.get_mac_info()
+        connection.get_product_info()
```

Comment:
Should we skip this if `uuid_q` is not empty?

Reply:
Agreed, we should I've made the change & added tests in https://github.com/home-assistant/core/pull/77856/commits/35a366dc698e044efe4482fe370eef84b3506be5